### PR TITLE
Ensure that datagateway tests will continue to work with new datagateway-api config

### PR DESCRIPTION
**Pull request by Reillyhewitson**
## Description
Datagateway-api will begin using a new configuration file soon. This pr ensures that the new configuration would be supported.
## Testing instructions
Ensure that you have checked out datagateway-api branch feature/use-yaml-config#372

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] Merge https://github.com/ral-facilities/datagateway-api/pull/388
- [ ] Change branch to checkout datagateway-api main.

## Agile board tracking
connect to #372

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -366,14 +345,15 @@ jobs:
...
+
       - name: Run datagateway-download e2e tests
         run: yarn workspace datagateway-download run e2e
       - name: Upload Cypress screenshots
         if: failure()
         uses: actions/upload-artifact@v3
         with:
           name: Download-Screenshots
-          path: packages/datagateway-download/cypress/screenshots
+          path: packages/datagateway-dataview/cypress/screenshots
```
Comment (louise-davies) [inline .github/workflows/ci-build.yml:356]: There's a copy paste typo here

File: .github/workflows/ci-build.yml [hunk 2]
```diff
@@ -524,11 +493,12 @@ jobs:
...
+
       - name: Run datagateway-search e2e tests
         run: yarn workspace datagateway-search run e2e
       - name: Upload Cypress screenshots
         if: failure()
         uses: actions/upload-artifact@v3
         with:
           name: Search-Screenshots
-          path: packages/datagateway-search/cypress/screenshots
+          path: packages/datagateway-dataview/cypress/screenshots
```
Comment (louise-davies) [inline .github/workflows/ci-build.yml:504]: There's a copy paste typo here

File: .github/workflows/ci-build.yml (hunk 1 above)
Comment (Reillyhewitson) [reply .github/workflows/ci-build.yml]: Fixed this in latest commit.

**Review by louise-davies — COMMENTED** [review]
This looks good to me :) Just need to not use the feature branch but otherwise take this as approval

**Comment by MRichards99** [comment]
Looks good, I know this wasn't the easiest thing to do so well done for solving it! As Louise says, once https://github.com/ral-facilities/datagateway-api/pull/388, just change the reference back

**Review by MRichards99 — APPROVED** [review]

**Pull request merged**